docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -154,7 +154,7 @@ For more information about OIDC authentication and authorization methods you can
 |====
 |OIDC topic |Quarkus information resource
 |Bearer Token authentication mechanism|xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer authentication]
-|Authorization Code Flow authentication mechanism|xref:security-openid-connect-web-authentication.adoc[OpenID Connect (OIDC) authorization code flow mechanism]
+|Authorization Code Flow authentication mechanism|xref:security-oidc-web-authentication-concept.adoc[Web authentication using OIDC authorization code flow]
```

Comment:
I'm not sure about `Web authentication using OIDC authorization code flow`, OIDC is what we use in the docs as an acronym, but the `Using OpenID Connect to Protect Web Applications` is precisely what the role of this code flow mechanism is. `Web authentication` is not really a correct term when it applies to Quarkus. Typing it in Google leads to WebAuthn docs among other things.

So please use the original `Using OpenID Connect to Protect Web Applications` or if you'd like, may be something like `OpenId Connect authorization code flow to protect web applications` ?